airbyte-integrations/connectors/source-mysql/src/main/java/io/airbyte/integrations/source/mysql/MySqlSourceOperations.java

```diff
@@ -111,14 +111,7 @@ public void copyToJsonField(final ResultSet resultSet, final int colIndex, final
       case DATETIME -> putTimestamp(json, columnName, resultSet, colIndex);
       case TIMESTAMP -> putTimestampWithTimezone(json, columnName, resultSet, colIndex);
       case TIME -> putTime(json, columnName, resultSet, colIndex);
-      case CHAR, VARCHAR -> {
-        if (field.isBinary()) {
-          // when character set is binary, the returned value is binary
-          putBinary(json, columnName, resultSet, colIndex);
-        } else {
-          putString(json, columnName, resultSet, colIndex);
-        }
-      }
+      case CHAR, VARCHAR -> putString(json, columnName, resultSet, colIndex);
```

Comment:
Do we need a test case for binary fields to ensure that conversion to string logic is taking place as expected?

Reply:
This code change is putting us back to handling VARCHAR the same way under any charset or collation.

We don't have tests for variation of charset as far as I can tell, but the change itself is returning to the way this used to behave before checking for `.isBinary()` was added.